ftw/theming/resources/scss/globals/variables.scss

```diff
@@ -198,6 +196,80 @@ $screen-XXS: "(min-width: #{$mobile-tiny})" !default;
   }
 }
 
+$primary-color: #205C90 !default;
```

Comment:
We could declare a variable above, e.g.
```sass
$include-deprecated-variables: false !default;
```

then we could put everything here in an `if`:
```sass
@if $include-deprecated-variables {
...
}
```
This would make it easier to find usages of old variables while still not breaking when deprecated variables were included.
What do you think about that idea?